src/adr/0012-jmap-partial-reads.md

```diff
@@ -0,0 +1,39 @@
+# 12. Projections for JMAP Messages
+
+Date: 2019-10-09
+
+## Status
+
+Accepted (lazy consensus)
```

Comment:
Is the status really Accepted? I mean, unless we have performance data, we can't really decide if it's good or not, right?